build.gradle

```diff
@@ -86,4 +86,152 @@ task loadReferenceDataStaging(type: com.marklogic.gradle.task.MlcpTask) {
   ]
 }
 
-mlLoadData.dependsOn loadReferenceDataStaging
\ No newline at end of file
+task loadMetaDataStaging(type: com.marklogic.gradle.task.MlcpTask) {
+  doFirst {
+    database = mlStagingDbName
+    port = mlStagingPort.toInteger()
+  }
+  classpath = configurations.mlcp
+  command = "IMPORT"
+  args = [
+      "-input_file_path", "$project.projectDir/src/main/ml-data/meta",
+      "-output_collections", "metaData",
+      "-output_permissions", "raw-data-reader,read,mes-writer,update",
+      "-document_type", "json",
+      "-output_uri_replace", ".*src/main/ml-data,''",
+      "-modules_root", "/",
+  ]
+}
+
+task loadMetaDataFinal(type: com.marklogic.gradle.task.MlcpTask) {
+  doFirst {
+    database = mlFinalDbName
+    port = mlFinalPort.toInteger()
+  }
+  classpath = configurations.mlcp
+  command = "IMPORT"
+  args = [
+      "-input_file_path", "$project.projectDir/src/main/ml-data/meta",
+      "-output_collections", "metaData",
+      "-output_permissions", "raw-data-reader,read,mes-writer,update",
+      "-document_type", "json",
+      "-output_uri_replace", ".*src/main/ml-data,''",
+      "-modules_root", "/",
+  ]
+}
+
+mlLoadData.dependsOn loadReferenceDataStaging
```

Comment:
This is a little unusual (adding our functions to an existing task) so let's add a comment. 

"To load additional reference and metadata, augment the existing mlLoadData task to look in these sub-directories and load them appropriately"